File: Data/DataTransferObjects/OrganizationDTO.cs
using System.Collections.Generic;
using Newtonsoft.Json;

namespace Fr8Data.DataTransferObjects
{
    public class OrganizationDTO
    {
        [JsonProperty("id")]
        public int Id { get; set; }

        [JsonProperty("name")]
        public string Name { get; set; }

        [JsonProperty("themeName")]
        public string ThemeName { get; set; }

        [JsonProperty("backgroundColor")]
        public string BackgroundColor { get; set; }

        [JsonProperty("logoUrl")]
        public string LogoUrl { get; set; }
    
    }
}

File: Hub/HubWeb/Scripts/app/model/OrganizationDTO.ts
module dockyard.model {
    export class OrganizationDTO {
        public id: number;
        public name: string;
        public themeName: string;
        public backgroundColor: string;
        public logoUrl: string;
        constructor(id: number, name: string, themeName: string, backgroundColor: string, logoUrl: string) {
            this.id = id;
            this.name = name;
            this.themeName = themeName;
            this.backgroundColor = backgroundColor;
            this.logoUrl = logoUrl;
        }
    }
    
}
 